packages/dashboard/src/components/ui/searchField.tsx

```diff
@@ -15,22 +15,33 @@ type SearchFieldProps = {
 const SearchField = ({ placeholder, onSearch, disabled }: SearchFieldProps) => {
```

Comment:
@agoldis just improved the search making sure the parameter is being removed when no search value